Readme.md

```diff
@@ -446,7 +446,13 @@ class UpdateUserEmailMutation extends Mutation
     {
         return [
             'id' => ['required'],
-            'email' => ['required', 'email']
+            'email' => ['required', 'email'],
+            'password' => function($arguments, $allArguments) {
```

Comment:
`$arguments, $allArguments`

I guess this comes from `\Rebing\GraphQL\Support\Rules::resolveRules`

Do we have better names for this (also through the code / tests), this is quite ambiguous to say at least.